src/workspaceViewer.ts

```diff
@@ -113,7 +111,7 @@ export function clearWorkspace(): void {
 
 export function saveWorkspace(): void {
 	void window.showSaveDialog({
-		defaultUri: Uri.file(getWorkspacePath() + path.sep + 'workspace'),
+		defaultUri: Uri.file(workingDir + path.sep + 'workspace'),
```

Comment:
Looks like it should be `'workspace.RData'`.

Reply:
I think it should save as workspace.RData because of `filters: {'Data': ['RData']}`

Reply:
I tried it on Ubuntu but no luck, it saves to workspace rather than workspace.RData.

Reply:
Ah, rats, thanks for the heads up! I'll fix that up

Reply:
eslint complains that "Operands of '+' operation must either be both strings or both numbers. Consider using a template literal.".